backend/src/nodes/impl/ncnn/model.py

```diff
@@ -704,7 +704,7 @@ def bin(self) -> bytes:
 
 class NcnnModelWrapper:
     def __init__(self, model: NcnnModel) -> None:
-        self.model: NcnnModel = model
+        self.model: NcannModel = model
```

Comment:
Uhhhh....

Reply:
Uhh, not sure how that happened.

Reply:
> How does this work now? You removed the dependency so how does it know to use the fork?

It doesn't...  I've just installed it manually for testing.  I'm not sure the best way to do this.  I think the dependency manager code needs to change a bit to accommodate installing from a github link.  Or maybe it's better to use a git submodule.

Reply:
I think I'd rather you just publish it to pypi and we install like normal. I forked NCNN and upload it to pypi under ncnn_vulkan, for example, so its not unprecedented